u8timeseries/utils/timeseries_generation.py

```diff
@@ -0,0 +1,137 @@
+import numpy as np
+import pandas as pd
+import math
+from pandas.tseries.offsets import DateOffset
+from pandas.tseries.frequencies import to_offset
+from pandas.tseries.holiday import USFederalHolidayCalendar
+
+from u8timeseries.timeseries import TimeSeries
+
+
+
+def constant_timeseries(value: float = 0, length: int = 10, offset: str = 'D',
```

Comment:
I would have recommended a non-zero default `value` here, so that it can be used out-of-the box as an external regressor in additive models (e.g. `value: float = 1.` would do I think).
Also, I would rename `offset` in `freq`, and maybe `start_date` in `start_ts` to reflect it needs not be an actual date?